app/models/deal.rb

```diff
@@ -0,0 +1,7 @@
+class Deal < ApplicationRecord
+  validates :name, :amount, presence: true, length: { maximum: 100 }
+  validates :name, :amount, :categories, presence: true, length: { maximum: 100 }
```

Comment:
- [ ] There are repetitions of name, and amount here, kindly remove the repetitions to have cleaner and more readable code.

Reply:
This issue is well resolved! ✔️